spelling/action.yml

```diff
@@ -20,8 +20,14 @@ runs:
     - name: Vale
       uses: errata-ai/vale-action@v2
       with:
-        files: '["./docs/", "./blog/"]' 
-        vale_flags: "--config ./.docs-gha/spelling/config/.vale.ini --glob='!docs/**/*.js'"        
+        files: |
+          "./docs/", 
+          "./blog/", 
+          "./services/", 
+          "./developer-tools/", 
+          "./snaps/", 
+          "./wallet/" 
+        vale_flags: "--config ./.docs-gha/spelling/.vale.ini --glob='!docs/**/*.js'"        
```

Comment:
@joshuafernandes I may have misread vale docs, but AFAIK the config folder is for vale-related config, not for the actual vale.ini -- I moved it up a level when fixing/testing

Reply:
This will work well too, tbh prob better coz its a lot more explicit as to where to get it from